tests/test_yaml_validation.py

```diff
@@ -36,6 +36,17 @@ def test_validate_yaml_empty_section(self):
         with self.assertRaises(ValidationError):
             validate_yaml(file_in)
 
+    def test_catch_invalid_time_block(self):
+        # time block without kew word "variable"
+        file_in = os.path.join(self.test_folder, 'ode_input_invalid_time_1.yaml')
+        with self.assertRaises(ValidationError):
+            validate_yaml(file_in)
+
+        # time block without kew word "variable"
+        file_in = os.path.join(self.test_folder, 'ode_input_invalid_time_2.yaml')
+        with self.assertRaises(ValidationError):
+            validate_yaml(file_in)
+
```

Comment:
I would split this into two tests, with the unittest package i think each test should test only one thing, that way when it fails you know exactly what failed.